ddtrace/internal/serverless/mini_agent.py

```diff
@@ -0,0 +1,38 @@
+import os
+from subprocess import Popen
+
+from . import in_gcp_function
+from ..compat import PYTHON_VERSION
+from ..logger import get_logger
+
+
+log = get_logger(__name__)
+
+
+def maybe_start_serverless_mini_agent():
+    if not in_gcp_function():
+        return
+
+    try:
+        python_version_list = PYTHON_VERSION.split(".")
+        rust_binary_path = (
+            "/layers/google.python.pip/pip/lib/python"
+            + python_version_list[0]
+            + "."
+            + python_version_list[1]
+            + "/site-packages/datadog-serverless-agent-linux-amd64/datadog-serverless-trace-mini-agent"
+        )
+
+        if os.getenv("DD_MINI_AGENT_PATH") is not None:
+            rust_binary_path = os.getenv("DD_MINI_AGENT_PATH")
+
+        log.debug(
+            "rust_binary_path: %s",
+            rust_binary_path,
+        )
```

Comment:
nit: this can be one line
```suggestion
        log.debug("rust_binary_path: %s", rust_binary_path)
```